src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -466,7 +466,7 @@
       (fn [{:keys [success? status-code get-parsed-response]}]
         (swap! state assoc :user-email (get-in (get-parsed-response) ["userInfo" "userEmail"]))
         (cond
-          (= status-code 502)
+          (contains #{0 502} status-code)
```

Comment:
Should be `contains?`, right?  Also, an idiomatic way to do a set membership check is to just use the set itself:

    (#{0 502} status-code)

But explicitly doing a `contains?` is fine too.